app/Livewire/Produit/Produit/ProduitShow.php

```
@@ -0,0 +1,205 @@
+                                                ->options(Entrepot::query()->where('status', 1)->pluck('name', 'id'))
+                                                ->required(),
+
+                                            TextInput::make('quantite')
+                                                ->label('Quantité')
+                                                ->default(0),
+                                        ])
+                                        ->columns(2),
+                                ]),
+                        ]),
+            ])
+            ->using(function (array $data) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$data'. (Unused Code Rules)

(UnusedFormalParameter)